lambdas/url_access_patterns/lambda_for_generateURL/index.js

```diff
@@ -46,6 +46,10 @@ exports.handler = async (event) => {
     TableName: "V-Transfer",
     Item: url,
   }
+  // if event.method is 'INVOKE' then dynamo call is not made and params is returned
+  if(event.method == 'INVOKE') {
+    return params
+  }
```

Comment:
Instead of returning query parameters that are to be executed in `fileUpload lambda`, can we do the put operation in this `generateURL lambda` and return only the `GS1_PK` when the `event.method == "Invoke"` .